Tasks/SonarQubePostTest/PRCA/PostComments-Module.psm1

```diff
@@ -225,6 +225,11 @@ function FilterMessagesByPath
 {
     param ([Array]$messages)
     
+    if (!(HasElements $messages))
```

Comment:
Somewhat unrelated, I've discovered a small bug in filtering which is easily fixed by exiting early